.github/agents/README.md

```
@@ -10,10 +18,26 @@ This directory contains instructions for AI coding agents working on this reposi
+pnpm run build     # Astro check + build
+pnpm run preview   # Preview production build
+pnpm run test      # Vitest unit tests
+pnpm run test:e2e  # Playwright E2E tests
+pnpm run lint      # Biome linting
+```
+
 ## Usage
 
 AI agents should reference these files for repository-specific guidance.
 
+### Key Documentation
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)